common/lib/common-utils/src/timer.ts

```diff
@@ -57,10 +57,41 @@ interface IRunningTimerState extends ITimeout {
     restart?: ITimeout;
 }
 
+const maxSetTimeoutMs = 0x7FFFFFFF; // setTimeout limit is MAX_INT32=(2^31-1).
+
+/**
+ * Sets timeouts like the setTimeout function allowing timeouts to exceed the setTimeout's max timeout limit.
+ * The longer the timeout, the less accurate the timeout is exactly going to be
```

Comment:
I am still a little confused about the second part of the comment. Why would it be less accurate?
Also, what is this trying to say? Is it a warning that longer timeouts are less accurate so be careful and account for it when handling the timeout?
I feel like it should say why its less accurate and then because its less accurate, what can the users do about it if they have to use it.

Reply:
I think this describes it better than how I can 
https://stackoverflow.com/questions/21097421/what-is-the-reason-javascript-settimeout-is-so-inaccurate

Reply:
Yeah, I think it is well understood that setTimeout is not accurate. We use it with that in mind, i.e., in places where we don't need precision. This is not specific to `setLongTimeout` function that you have added here so this comment doesn't belong here.

Reply:
The updated comment looks fine to me. It says that timeouts in general are not accurate.